local-volume/test/e2e/glide.yaml

```diff
@@ -0,0 +1,40 @@
+package: github.com/kubernetes-incubator/external-storage/local-volume/test/e2e
```

Comment:
I would prefer if we did not check in local-volume/test/e2e/vendor (add it to .gitignore & `make clean`?) simply because cloners will most likely not need it and it's huge. 

For the NFS e2e tests, I do glide install -v every time before I run the e2e tests, can we do something like that? If the vendor directory already exists, it takes a couple of second for glide to verify else it pulls it on-demand